daml/ContingentClaims/Lifecycle.daml

```diff
@@ -90,6 +90,9 @@ acquire' spot (Cond obs c c') = do
 acquire' spot (Anytime obs c) = do
   predicate <- compare spot obs
   pure $ AnytimeF obs if predicate then Right c else Left c
+acquire' spot (Until obs c) = do
```

Comment:
The reason this wasn't done before was because we relied on `acquire` below to traverse `Until` eagerly (see below).